torch/csrc/dynamo/guards.cpp

```diff
@@ -128,6 +133,11 @@ class TensorCheck {
         return fail_reason.str();
       }
     }
+    if (offset_.has_value() && offset_.value() != v.storage_offset()) {
+      fail_reason << "storage_offset mismatch, expected " << v.storage_offset()
```

Comment:
expected and actual are reversed?

Reply:
yea

Reply:
Good catch

Reply:
Oh I think it was wrong for sizes and strides too.